src/scenario-registry.ts

```diff
@@ -175,6 +178,14 @@ export const scenarios = [
     displayOrder: 1,
     rootComponent: SubscribeToSubscriptionPlan,
   },
+
+  {
+    groupName: 'Billing & Monetization',
+    shortId: 'list-user-subscriptions',
+    displayName: 'List User Subscriptions',
+    displayOrder: 2,
```

Comment:
this scenario we could position as the 1st one?

Reply:
Isn't it better to have "List Subscription Plans Anonymously" scenario as the first one, and this one as the second?